types/setting.go

```diff
@@ -1052,14 +1055,30 @@ var (
 	SettingDefinitionOrphanAutoDeletion = SettingDefinition{
 		DisplayName: "Orphan Auto-Deletion",
 		Description: "This setting allows Longhorn to delete the orphan resource and its corresponding orphaned data automatically. \n\n" +
-			"Orphan resources on down or unknown nodes will not be cleaned up automatically. \n\n",
+			"Orphan resources on down or unknown nodes will not be cleaned up automatically. \n\n" +
```

Comment:
Mention the setting is deprecated and replaced by the new setting in the description.

cc @sushant-suse

Reply:
Is it just setting rename?

Reply:
Oh. I see. New setting can control multiple orphan types